portal/src/main/webapp/js/src/crosscancer.js

```diff
@@ -1444,5 +1590,45 @@
         Backbone.history.start();
     });
 
+
 })(window.jQuery, window._, window.Backbone, window.d3);
 
+
+///////////////jiaojiao added 
+var PanCancerTemplateCache = (function () {
```

Comment:
it looks like you are not using this block anywhere in this pull request. So you could remove this code.